tests/functional/test_plugin.py

```diff
@@ -202,7 +202,7 @@ def test_plugin_slowinit(node_factory):
     n = node_factory.get_node()
 
     with pytest.raises(RpcError, match='failed to respond to \'init\' in time, terminating.'):
-        n.rpc.plugin_start(os.path.join(os.getcwd(), "tests/plugins/slow_init.py"))
+        n.rpc.plugin_start(os.path.join(os.getcwd(), "tests/functional/plugins/slow_init.py"))
```

Comment:
moving all existing tests seems incredibly unnecessary. what's wrong with having a `fuzzing` directory in along with all the other functional tests?

Reply:
Yeah it might seem overkill but it looked like the right thing to me: `tests` right now contains our functional tests, and mixing up fuzz testing and functional tests seems to be a mess..

Reply:
You're not mixing them though, the fuzz tests all live in `tests/fuzz`